### PR TITLE
Update Ember

### PR DESCRIPTION
**Pull request by RobbieTheWagner**
Initial PR to update some outdated stuff before trying to remove jQuery.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,54 +2,60 @@
...
+  "dependencies": {
+    "ember-cli-babel": "^6.8.1"
   },
-  "author": "",
-  "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.0.2",
-    "ember-cli": "2.12.1",
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-cli": "~2.15.0-beta.2",
```
Comment (YoranBrondsema) [inline package.json:37]: Probably doesn't matter too much, but is it sufficient to use a stable release instead of a beta release?

**Comment by YoranBrondsema** [comment]
Great cleanup, thanks! Makes me think we should set up Travis CI on this repo. I'll try later today if I can (I'm not the main maintainer of the repo).

File: package.json (hunk 1 above)
Comment (RobbieTheWagner) [reply package.json:37]: It really doesn't make a difference for addons. Only things in dependencies have any effect on the consuming application. devDependencies are just for developing the addon.

**Comment by RobbieTheWagner** [comment]
I would be glad to setup Travis, if you wanted to make me a collaborator.

**Comment by YoranBrondsema** [comment]
Alright :-) I just checked and I can't seem to add you. Maybe @briangonzalez can help us with that?

**Comment by briangonzalez** [comment]
Good to go :)

**Comment by RobbieTheWagner** [comment]
🎉 thanks @briangonzalez!

**Pull request merged**